test/unit/conftest.py

```diff
@@ -29,30 +30,36 @@ class AutosubmitExperiment:
 
 
 @pytest.fixture(scope='function')
-def autosubmit_exp(autosubmit: Autosubmit, request: pytest.FixtureRequest) -> Callable:
+def autosubmit_exp(
+        autosubmit: Autosubmit,
+        request: pytest.FixtureRequest) -> Callable:
     """Create an instance of ``Autosubmit`` with an experiment."""
 
     original_root_dir = BasicConfig.LOCAL_ROOT_DIR
     tmp_dir = TemporaryDirectory()
     tmp_path = Path(tmp_dir.name)
 
     def _create_autosubmit_exp(expid: str):
-        # directories used when searching for logs to cat
         root_dir = tmp_path
         BasicConfig.LOCAL_ROOT_DIR = str(root_dir)
-        exp_path = root_dir / expid
-        exp_tmp_dir = exp_path / BasicConfig.LOCAL_TMP_DIR
-        aslogs_dir = exp_tmp_dir / BasicConfig.LOCAL_ASLOG_DIR
+        exp_path = BasicConfig.expid_dir(expid)
+
+        # directories used when searching for logs to cat
+        exp_tmp_dir = BasicConfig.expid_tmp_dir(expid)
+        aslogs_dir = BasicConfig.expid_aslog_dir(expid)
         status_dir = exp_path / 'status'
-        aslogs_dir.mkdir(parents=True, exist_ok=True)
-        status_dir.mkdir(parents=True, exist_ok=True)
+        os.makedirs(aslogs_dir, exist_ok = True)
+        os.makedirs(status_dir, exist_ok = True)
```

Comment:
Sorry, I think I said to use the `exist_ok`, but now I realize we could also drop the `os` module here.

```
        aslogs_dir.mkdir(parents=True, exist_ok=True)
        status_dir.mkdir(parents=True, exist_ok=True)
```

I think that should work the same way, and then we can also remove the `import os` at the top.